server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java

```diff
@@ -380,49 +384,56 @@ private static int audit(final CountyDashboard cdb,
     final Set<AuditReason> disagreements = new HashSet<>();
     final CastVoteRecord cvr_under_audit = auditInfo.cvr();
     final CastVoteRecord audit_cvr = auditInfo.acvr();
+    int totalCount = 0;
 
     for (final CVRContestInfo ci : audit_cvr.contestInfo()) {
       if (ci.consensus() == ConsensusValue.NO) {
         contest_disagreements.add(ci.contest().name());
       }
     }
 
-    final int audit_count = auditInfo.multiplicity() - auditInfo.counted();
-
     for (final ComparisonAudit ca : cdb.comparisonAudits()) {
+      // FIXME extract-fn: multiplicityFu
+      final int multiplicity = ca.multiplicity(auditInfo.cvr().id());
+      final int auditCount = multiplicity - auditInfo.getCountByContest(ca.id());
+      totalCount += auditCount;
 
-      final OptionalInt discrepancy = ca.computeDiscrepancy(cvr_under_audit, audit_cvr);
+      auditInfo.setMultiplicityByContest(ca.id(), multiplicity);
+      auditInfo.setCountByContest(ca.id(), multiplicity);
```

Comment:
This is supposed to be `multiplicity` and not `auditCount`, right? I'm just double-checking based on names, not intimate knowledge of the code.